README.md

```diff
@@ -29,14 +29,12 @@
 
 ![Architecture](https://user-images.githubusercontent.com/52545545/147286088-08b0d11f-d81d-4622-8145-179071d2f0fb.png)
 
-# Table of Contents
-* [What is Firezone?](#what-is-firezone)
-  * [Features](#features)
-  * [Anti-features](#anti-features)
-* [Documentation](#documentation)
-* [Get Help](#get-help)
-* [Developing and Contributing](#developing-and-contributing)
-* [Security](#security)
+# Get started
+
+Follow our installation guide at https://docs.firez.one/get-started to install your self-hosted instance of Firezone. 
```

Comment:
Perhaps you meant https://docs.firez.one/docs/get-started? This link 404s

Reply:
fixed